### DISCUSSION TITLE
Show second level section in Left Sidebar - HTML Format

### DISCUSSION THREAD

**Discussion by fulem** [quarto-dev/quarto-cli#4134] [Q&A]
I would like the HTML left sidebar to show also the second level section collapsible, like
```
Section 1 >
  1.1 A
  1.2 B
  ...
Section 2 >
  2.1 C
  2.2 D
  ...
```
Currently i only get the first level sections, the rest appears in the table of contents at the right. Is something like this possible?. 

According to the docs the HTML format should support more or less the same functionality as the Website format which allows to create arbitrary hierarchies in the left sidebar. However, the `contents` attribute in `sidebar` is ignored.

An alternative to this would be numbering sections to depend on top level parts. For example:
```
Part 1 
  1.1 The first section of Part 1
  1.2 The second section of Part 2
  ...
Part 2
  2.1 The first section of Part 2
  2.2 The second section of Part 2
  ...
```
which is more or less the same thing i mentioned earlier but using parts as top level.
Is this possible?.

**Comment by mcanouil** [comment]
Without an actual example of a Quarto document, I am not completely sure what you are talking about ...

See <https://quarto.org/docs/reference/formats/html.html#table-of-contents> for the `html` format options related to table of contents.
